### ISSUE TITLE
Standard laser modules giving bonus to Pulse lasers

### ISSUE THREAD

**Issue by TheGreenHell** [EmilyBjoerk/lsml#502]
LSML version: 1.7.2

lsml://rgACBiwEMBI6BiwEMAbnf6/upzrMsx5jxiq2xs0dVbb+efPU102c2U3/5BI=

Steps to reproduce:
1. Open lsml.
2. Create mech using a standard laser (any type).
3. Apply cooldown module for that laser.
4. Swap laser for pulse version.
5. Apply cooldown module for pulse version.

Actual result:  Cooldowns are affected by both modules for the pulse lasers (which also affects DPS and heat results as well).

Expected result:  Pulse lasers should not be recieving benefit from standard laser cooldown modules.

Edit:  This also works with range modules.  LSML stacks them, but in game pulses gain no benefit from standard laser range modules.

Notes:  I tried this as an experiment to see if lasers work the same as PPC's as far as quirks are concerned (in game ERPPC's recieve benefit from standard PPC modules and quirks).  Turns out, they do not work that way as in game my LPL's keep the 2.8s cooldown, and not the 2.4s cooldown that LSML says they should have.  Confirmed in game using the build posted above (Took 3 volleys to get LPL's .5 seconds ahead of ML's).  I know this might be a small detail that most people wouldn't run into, but I figured I should mention it anyways.

**Comment by EmilyBjoerk** [comment]
I'm currently away and cannot check this but I know that testing grounds
doesn't (at least didn't a month ago) take the laser duration into account.

If you tested on testing grounds, could you please retest in an actual game
or lobby match?

**Comment by TheGreenHell** [comment]
Aye aye Capitan!  Testing as we speak.

**Comment by TheGreenHell** [comment]
Confirmed using a macro with accuracy down to 1 millisecond.  The standard laser modules do nothing for their pulse and ER brothers.

I used an LPL with both LL cd% and LPL cd% modules on.  I wrote a macro for it with a timing of 1180ms per shot (the cycle time if both mods were working).  The LPL was on cooldown for an extra second while it waited for the macro to cycle.  Re-wrote the macro set to 1570 (the normal cycle time with LPL cd% module only) and it fired with perfect timing, no lag at all.

Also confirmed in game with range modules.  The standard laser range modules do not increase the range of pulses and ER's at all.  This was easy to confirm since it shows the range on screen.
